doc/installation/docker.rst

```diff
@@ -41,20 +109,20 @@ This command will:
 - Map port 8000 on the host to port 8000 in the container (`-p 8000:8000`)
 - Name the container "edumfa" (`--name edumfa`)
 
-Persistent Data
+Persistent Data 
 ...............
 
 To persist data between container restarts, you can mount a volume for the database:
 
 .. code-block:: bash
 
-   docker run -d -p 8000:8000  -v /path/to/config.cfg:/etc/edumfa/edumfa.cfg -v edumfa_data:/etc/edumfa --name edumfa ghcr.io/edumfa/edumfa:latest
+   docker run -d -p 8000:8000  -v /path/to/edumfa.cfg:/etc/edumfa/edumfa.cfg -v edumfa_data:/etc/edumfa --name edumfa ghcr.io/edumfa/edumfa:latest
 
 This will create a named volume `edumfa_data` that will persist your eduMFA data. This volume will contain the encryption key and the audit key.
```

Comment:
```suggestion
This will create a named volume `edumfa_config` that will persist your eduMFA configuration. This volume will contain the encryption key and the audit key.
```